modules/ebi-metagenomics/seqfu/check/tests/nextflow.config

```diff
@@ -0,0 +1,7 @@
+process{
+
+    withName: SEQFU_CHECK {
+        ext.args = '-s -d'
```

Comment:
Maybe we should "hardcode" `-s`... just to avoid accidental issues if some forgets to load this file... happy to chat about this